src/kartoffel/kartoffel.interface.ts

```diff
@@ -59,3 +59,45 @@ export interface IKartoffelUser {
     // Calculated
     fullName?: string;
 }
+
+export interface IDigitalIdentity {
+    type: string;
+    source: string;
+    mail: string;
+    uniqueId: string;
+    entityId: string;
+    createdAt: Date;
+    updatedAt: Date;
+    isRoleAttachable: Boolean;
+  }
+  
+  export interface IKartoffelUserNew {
```

Comment:
remove & replace the old interfaces, it will be deprecated.

Reply:
Done.